tests/integration/clarity/natives/get-burn-block-info.spec.ts

```diff
@@ -0,0 +1,240 @@
+    chainUpdate = await waitForNextPreparePhase(network, orchestrator, -2);
+    let blockHeight = getBitcoinBlockHeight(chainUpdate);
+
+    // Broadcast some STX stacking orders
+    let response = await broadcastStackSTX(
+      2,
+      network,
+      25_000_000_000_000,
+      Accounts.WALLET_1,
+      blockHeight,
+      12,
+      1000
+    );
+    expect(response.error).toBeUndefined();
+
+    // Wait for block N+1 where N is the height of the next reward phase
+    chainUpdate = await waitForNextRewardPhase(network, orchestrator, 1);
+    let height = getBitcoinBlockHeight(chainUpdate);
+
+    for (let index = 0; index < 12; index++) {
+      orchestrator.waitForBitcoinBlock();
+      // Build a transaction to call the contract
+      let callTxOptions: SignedContractCallOptions = {
+        senderKey: Accounts.WALLET_1.secretKey,
+        contractAddress: Accounts.DEPLOYER.stxAddress,
+        contractName: "test-2-1",
+        functionName: "test-2",
+        functionArgs: [uintCV(height + index)],
+        fee: 2000,
+        network,
+        anchorMode: AnchorMode.OnChainOnly,
+        postConditionMode: PostConditionMode.Allow,
+      };
+      let transaction = await makeContractCall(callTxOptions);
+
+      // Broadcast transaction
+      let result = await broadcastTransaction(transaction, network);
+      console.log("RESULT", <TxBroadcastResultOk>result);
+      expect((<TxBroadcastResultOk>result).error).toBeUndefined();
+
+      let callPoxTxOptions: SignedContractCallOptions = {
+        senderKey: Accounts.WALLET_1.secretKey,
+        contractAddress: "ST000000000000000000002AMW42H",
+        contractName: "pox-2",
+        functionName: "get-stacker-info",
+        functionArgs: [principalCV(Accounts.WALLET_1.stxAddress)],
+        fee: 2000,
+        network,
+        anchorMode: AnchorMode.OnChainOnly,
+        postConditionMode: PostConditionMode.Allow,
+      };
+      transaction = await makeContractCall(callPoxTxOptions);
+
+      // Broadcast transaction
+      result = await broadcastTransaction(transaction, network);
+      console.log("RESULT", <TxBroadcastResultOk>result);
+
+      // expect((<TxBroadcastResultOk>result).error).toBeUndefined();
+
+      // Wait for the transaction to be processed
+      let [_, tx] = waitForStacksTransaction(
+        orchestrator,
+        Accounts.WALLET_1.stxAddress
+      );
+      console.log("HEIGHT:", height + index);
+      console.log("=== ", tx.result);
+
+      // expect(tx.description).toBe(
+      //   `invoked: ${Accounts.DEPLOYER.stxAddress}.test-2-1::test-2(u${height+ index})`
+      // );
+      // expect(tx.result).toBe(
+      //   "(ok (some (tuple (addrs ((tuple (hashbytes 0x0000000000000000000000000000000000000000) (version 0x00)) (tuple (hashbytes 0x0000000000000000000000000000000000000000) (version 0x00)))) (payout u10000))))"
+      // );
+      // expect(tx.success).toBeTruthy();
```

Comment:
is there anything wrong with these assertions?

Reply:
Oh, right. I forgot to note that this test is still not functional, due to the pox-2 contract not showing up. There is a bunch of debug stuff still in there.

Reply:
I'll clean it back up to what should pass once everything is working.

Reply:
@lgalabru do you see anything wrong with this test otherwise? I'd like to see wallet 1 in the `pox-addrs` output, but it has not been working. I believe the two addresses that I'm getting are just burn addresses. Is that right?

Reply:
hmm I 'll look into it.